passl/engine/trainer.py

```diff
@@ -189,7 +189,7 @@ def __init__(self, cfg):
             scale_loss = amp_cfg.pop('scale_loss')
             self.scaler = paddle.amp.GradScaler(init_loss_scaling=scale_loss)
             amp_cfg['models'] = self.model
-            self.model = paddle.amp.decorate(**amp_cfg)  # decorate for level O2
+            self.model, _ = paddle.amp.decorate(**amp_cfg)  # decorate for level O2
```

Comment:
Amp decorate in paddle 2.2.2 would return model and optimizer, but return model when input optimizer is none in paddle 2.3.0+, we would add paddle version check script later.